### PR TITLE
DFPL-2178: Fix NPE in about to submit callback for manage orders flow

### PR DESCRIPTION
**Pull request by qasimnawaz-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-2178

### Change description ###

Fix NPE in about to submit callback for manage orders flow caused by trying to access a field in a null allocatedJudge object

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java [hunk 1]
```diff
@@ -31,11 +31,10 @@ public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalA
...
                                                         Judge allocatedJudge) {
-        if (isEmpty(judgeAndLegalAdvisor) || judgeAndLegalAdvisor.isUsingAllocatedJudge()) {
-            return migrateJudgeAndLegalAdvisor(judgeAndLegalAdvisor, allocatedJudge);
-        } else {
-            return judgeAndLegalAdvisor;
-        }
+        return (isEmpty(judgeAndLegalAdvisor) ||
+                (judgeAndLegalAdvisor.isUsingAllocatedJudge() && !isEmpty(allocatedJudge)))
+            ? migrateJudgeAndLegalAdvisor(judgeAndLegalAdvisor, allocatedJudge)
+            : judgeAndLegalAdvisor;
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java:37]: Can we add a test for this edge case?
